<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="IN-OLX-PF2SH88V" failures="1" tests="2" name="Default test(failed)(failed)" time="6.769" errors="0" timestamp="2024-07-27T12:39:29 IST">
  <testcase classname="RetryWithXML.Test_Exucution" name="TestMethod2" time="3.31"/>
  <testcase classname="RetryWithXML.Test_Exucution" name="TestMethod3" time="0.002">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at RetryWithXML.Test_Exucution.TestMethod3(Test_Exucution.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TestMethod3 -->
  <testcase classname="RetryWithXML.Test_Exucution" name="TestMethod1" time="0.0">
    <ignored/>
  </testcase> <!-- TestMethod1 -->
</testsuite> <!-- Default test(failed)(failed) -->
